credentials/build.gradle.kts

```diff
@@ -8,6 +8,7 @@ repositories {
 
   maven("https://jitpack.io")
   maven("https://repo.danubetech.com/repository/maven-public/")
+  maven("https://repository.jboss.org/nexus/content/repositories/thirdparty-releases/")
```

Comment:
the dep we're pulling from here isn't available thru maven central? asking bc it becomes another repo we have to ask consumers to include in order to use our lib. we already have 2.

Reply:
it's not unfortunately. this is the most widely used lib to provide what we need (bencode) so I prioritized support; I had tried a few others but they didn't work as well.

during jar-ificiation it shouldn't matter much; if we run into consistency issues I can try to find another approach